todo-app/src/App.js

```diff
@@ -1,21 +1,56 @@
-import React, { Component } from 'react';
+import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import Todo from './Todo';
 
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
+  class App extends React.Component {
+    constructor (props){
+        super(props);
+        this.state = {
+          todos: [
+               {
+                description: "Todo List",
+                done: false
+               }
+            ]
+        };
+        
+    }
+    
+    handleChange = (e) => {
+    const description =  this.name.value;
+    const newToDo = {description};
+    const todosList = [...this.state.todos, newToDo];
+    this.setState({todos:todosList})
+  }
+    
+  
+   
+  onDelete= (index) => {
+    const todosList = [...this.state.todos];
+    todosList.splice(index, 1);
+    this.setState({todos:todosList});
+  }
+    render() {
+      let todoList=this.state.todos;
+      todoList= todoList.map(function(item,index){
```

Comment:
Although using `function (item, index){}.bind(this)` works, doesn't it become more readable when using an arrow function, for example `(item, index) => {}`? What do you think?